docs/source/torch.compiler_ipex.rst

```diff
@@ -0,0 +1,163 @@
+Intel® Extension for PyTorch* Backend
+=====================================
+
+To work better with `torch.compile`, Intel® Extension for PyTorch* implements a backend `ipex`. It targets to improve hardware resource usage efficiency on Intel platforms for better performance.
+
+Currently the `ipex` backend is implemented with two compilation paths: `TorchScript` path, and `TorchInductor` path. The former path uses `torch.jit.trace` and `torch.jit.freeze` to compile models. The latter path uses `inductor` with further customizations designed in Intel® Extension for PyTorch* for the model compilation. While the `inductor` backend is getting mature, the `TorchScript` path is used as the default path for the `ipex` backend. An API, `ipex._set_compiler_backend("inductor")`, is exposed to turn on the experimental TorchInductor path.
+
+Same as the functionality of `TorchScript` and `TorchInductor`, the `TorchScript` path optimizes for inference workloads only, while the `TorchInductor` path optimizes for both training and inference workloads.
+
+Usage Example
+~~~~~~~~~~~~~
+
+Train FP32
+----------
+
+.. code:: python
+
+   import torch
+   import torchvision
+
+   LR = 0.001
+   DOWNLOAD = True
+   DATA = 'datasets/cifar10/'
+
+   transform = torchvision.transforms.Compose([
+     torchvision.transforms.Resize((224, 224)),
+     torchvision.transforms.ToTensor(),
+     torchvision.transforms.Normalize((0.5, 0.5, 0.5), (0.5, 0.5, 0.5))
+   ])
+   train_dataset = torchvision.datasets.CIFAR10(
+     root=DATA,
+     train=True,
+     transform=transform,
+     download=DOWNLOAD,
+   )
+   train_loader = torch.utils.data.DataLoader(
+     dataset=train_dataset,
+     batch_size=128
+   )
+
+   model = torchvision.models.resnet50()
+   criterion = torch.nn.CrossEntropyLoss()
+   optimizer = torch.optim.SGD(model.parameters(), lr = LR, momentum=0.9)
+   model.train()
+
+   #################### code changes ####################
+   import intel_extension_for_pytorch as ipex
+   model, optimizer = ipex.optimize(model, optimizer=optimizer)
+
+   ipex._set_compiler_backend("inductor")
+   # dynamic mode: dynamic=True
+   # static mode: dynamic=False
+   compile_model = torch.compile(model, dynamic=False, backend="ipex")
```

Comment:
The IPEX backend supports both dynamic and static modes. I think the comments is unnecessary, and dynamic=False can be removed.